iree/samples/simple_embedding/CMakeLists.txt

```diff
@@ -66,26 +66,30 @@ if(NOT CMAKE_SYSTEM_PROCESSOR MATCHES "amd64|x86_64|AMD64")
   return()
 endif()
 
-iree_cc_test(
+if(${IREE_TARGET_BACKEND_DYLIB-LLVM-AOT} AND ${IREE_HAL_DRIVER_DYLIB})
```

Comment:
this condition is already checked by 

```
if(NOT ${IREE_TARGET_BACKEND_DYLIB-LLVM-AOT} OR NOT ${IREE_TARGET_BACKEND_VULKAN-SPIRV}
   OR NOT ${IREE_HAL_DRIVER_DYLIB} OR NOT ${IREE_HAL_DRIVER_VULKAN})
  return()
endif()
```

above. I think it's fine to keep or omit these redundant checks. As we get further into multiarch/heterogeneous execution, we can change "simple_embedding" to just use a single backend, then add a separate sample showing those features.

Reply:
Done.